### DISCUSSION TITLE
I accidentally registered as a student

### DISCUSSION THREAD

**Discussion by PatriciaMouraCruz** [community/community#133377] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I accidentally registered as a student but I'm actually a teacher. What could I do to change my licence to a Teacher.
Thank you!

**Comment by manzx666** [comment]
response#1

**Reply by aiman-tzy** [reply]
wow

**Comment by Sachinsingh2002** [comment]
hii @PatriciaMouraCruz there is no way you can change your licence, you have to contact Github support and by following these steps you can resolve your problem : 


1.Go to GitHub Support: Visit the (https://support.github.com/).
2.Submit a Request: Click on "Contact us" or "Submit a request".
3.Fill Out the Form: Fill out the form with your information and describe the issue. Be clear that you registered as a student by mistake and need to change your license to reflect your status as a teacher.
4.Submit the Request: Submit the form and wait for a response from GitHub Support.

**Reply by PatriciaMouraCruz** [reply]
Thank you so much for your help!

**Reply by Sachinsingh2002** [reply]
Happy to help!
